templates/pages/accessibilite.html

```diff
@@ -0,0 +1,68 @@
+{% extends "pages/base.html" %}
+
+
+
+{% block title %}
+    Déclaration d'accessibilité
+{% endblock %}
+
+
+
+{% block breadcrumb %}
+    <li>Déclaration d'accessibilité</li>
+{% endblock %}
+
+
+
+{% block headline %}
+    <h1>Déclaration d'accessibilité</h1>
+{% endblock %}
+
+
+
+{% block content %}
+    <p>L’association Zeste de Savoir s’engage à rendre son site internet accessibles conformément à
+        <a href="https://www.legifrance.gouv.fr/loda/article_lc/LEGIARTI000037388867" rel="nofollow">l’article 47 de la loi №&nbsp;2005-102 du 11 février 2005</a>.</p>
+
+    <p>
+        À cette fin, une attention est portée à l’accessibilité des mises à jour effectuées depuis décembre 2020 sur le site, date d'une reprise graphique de ce dernier, et des efforts de corrections ont été réalisés, parfois en coopération avec des personnes concernées volontaires.
+    </p>
+
+    <p>
+        Cette déclaration d’accessibilité s’applique au site web de Zeste de Savoir (<code class="notranslate">zestedesavoir.com</code>).
+    </p>
+
+    <h3>État de conformité</h3>
+    <p>
+        Aucun audit n’a été encore effectué.
+Le site web Zeste de Savoir est non-conforme avec le référentiel général d’amélioration de l’accessibilité (RGAA).
+    </p>
+    <p>
+        Cependant, un audit de conformité est prévu, bien qu’aucune date ne soit connue aujourd’hui.
+    </p>
+
+    <h3>Retour d’information et contact</h3>
+
+    <p>Si vous n’arrivez pas à accéder à un contenu ou à un service, vous pouvez contacter le responsable du site internet ou son équipe de développement, pour être orienté·e vers une alternative accessible ou obtenir le contenu sous une autre forme.</p>
+
+    <ul>
+        <li>Contacter le Conseil d’Administration de l’association Zeste de Savoir, à l’adresse <a href="mailto:association+accessibilite@zestedesavoir.com">association+accessibilite@zestedesavoir.com</a> ;</li>
+        <li>Contacter l’équipe de développement de Zeste de Savoir <a href="https://github.com/zestedesavoir/zds-site/issues/new">sur GitHub</a>.</li>
```

Comment:
```suggestion
        <li>Contacter le Conseil d’Administration de l’association Zeste de Savoir, à l’adresse <a href="mailto:association+accessibilite@zestedesavoir.com">association+accessibilite@zestedesavoir.com</a> ;</li>
        <li>Contacter l’équipe de développement de Zeste de Savoir <a href="https://github.com/zestedesavoir/zds-site/issues/new">sur GitHub</a> ou par courriel à l’adresse <a href="mailto:technique+accessibilite@zestedesavoir.com">technique+accessibilite@zestedesavoir.com</a>.</li>
```

Une proposition, pour aller plus vite (mais ça vaut ce que ça vaut)

Reply:
J'ai du mal à voir l'intérêt de mettre un contact par courriel à la fois vers l'équipe technique et le conseil d'administration de l'association. De mon point de vue, ça concerne principalement l'équipe technique donc je propose une formulation du type "Contacter l'équipe technique : - par courriel - sur Github"

Reply:
J'y vois une différence : l'équipe technique pour les questions techniques, et le CA pour les questions d'intention ou de responsabilité. Mais peut-être vais-je chercher trop loin ?

Il est d'usage de laisser un contact vers læ responsable ou l'équipe responsable. Il est en réalité plus rare de donner un contact vers l'équipe technique (mais c'est lié à l'organisation usuelle en entreprise où cette dernière n'est souvent pas autonome dans ses décisions…).

Reply:
Ajouté!